Enable wxWindow::SetBackgroundColour() for certain controls
(code blatantly copied from src/univ/stattext.cpp)

Signed-off-by: Anders Larsen <al@alarsen.net>
---
 src/univ/checkbox.cpp |    6 ++++++
 src/univ/radiobut.cpp |    6 ++++++
 2 files changed, 12 insertions(+)

Index: b/src/univ/checkbox.cpp
===================================================================
--- a/src/univ/checkbox.cpp
+++ b/src/univ/checkbox.cpp
@@ -149,6 +149,12 @@ void wxCheckBox::DoDraw(wxControlRendere
     int flags = GetStateFlags();
 
     wxDC& dc = renderer->GetDC();
+    if ( UseBgCol() )
+    {
+        dc.SetBrush(GetBackgroundColour());
+        dc.SetPen(*wxTRANSPARENT_PEN);
+        dc.DrawRectangle(renderer->GetRect());
+    }
     dc.SetFont(GetFont());
     dc.SetTextForeground(GetForegroundColour());
 
Index: b/src/univ/radiobut.cpp
===================================================================
--- a/src/univ/radiobut.cpp
+++ b/src/univ/radiobut.cpp
@@ -157,6 +157,12 @@ wxSize wxRadioButton::GetBitmapSize() co
 void wxRadioButton::DoDraw(wxControlRenderer *renderer)
 {
     wxDC& dc = renderer->GetDC();
+    if ( UseBgCol() )
+    {
+        dc.SetBrush(GetBackgroundColour());
+        dc.SetPen(*wxTRANSPARENT_PEN);
+        dc.DrawRectangle(renderer->GetRect());
+    }
     dc.SetFont(GetFont());
     dc.SetTextForeground(GetForegroundColour());
 

 	  	 
